x-pack/test/rbac_api_integration/apis/privileges/index.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from 'expect.js';
+
+export default function ({ getService }) {
+  describe('privileges', () => {
+    it(`get should return privileges`, async () => {
+      const supertest = getService('supertest');
+
+      await supertest
+        .get(`/api/security/v1/privileges`)
+        .expect(200)
+        .then(resp => {
+          expect(resp.body).to.eql([
+            {
+              application: 'kibana-.kibana',
+              name: 'all',
+              actions: ['version:7.0.0-alpha1', 'action:*'],
```

Comment:
Won't this version identifier being hardcoded be a problem? I don't see where this is being seeded from.

Reply:
Good catch, it'll definitely fail, I'll get it pulling from the right place.